src/content/developers/docs/nodes-and-clients/index.md

```diff
@@ -93,7 +95,7 @@ If you run a full node, the whole Ethereum network benefits from it.
 
 ## Alternatives {#alternatives}
 
-Running your own node can be difficult and you don’t always need to run your own instance. In this case, you can use a third party API provider like [Infura](https://infura.io), [Alchemy](https://alchemyapi.io), or [QuikNode](https://www.quiknode.io). Alternatively [ArchiveNode](https://archivenode.io/) is a community-funded Archive node that hopes to bring archive data on the Ethereum blockchain to independent developers who otherwise couldn't afford it.
+Running your own node can be difficult and you don’t always need to run your own instance. In this case, you can use a third party API provider like [Infura](https://infura.io), [Alchemy](https://alchemyapi.io), or [QuikNode](https://www.quiknode.io). Alternatively [ArchiveNode](https://archivenode.io/) is a community-funded Archive node that hopes to bring archive data on the Ethereum blockchain to independent developers who otherwise couldn't afford it. For overview of using these services, refer to [Nodes as a services article](/en/developers/docs/nodes-and-clients/nodes-as-a-service/).
```

Comment:
We don't need to label the links as an article. 

For an overview of using these services, check out [nodes as a services](/en/developers/docs/nodes-and-clients/nodes-as-a-service/)